### DISCUSSION TITLE
Error with uploading files with FileInput

### DISCUSSION THREAD

**Discussion by samuelBereznak** [trussworks/react-uswds#2677] [Q&A]
I am getting an error when I try to use the FileInput component. 
``react-dom.development.js:12056  Uncaught DOMException: Failed to execute 'readAsDataURL' on 'FileReader': The object is already busy reading Blobs.``
I only of one instance of FileInput yet this error points to multiple uses, being used at one time. Any help would be appreciated.

**Comment by werdnanoslen** [comment]
@samuelBereznak have you worked this out? If not, please share what you've tried so far to fix it and how you've implemented the component in your code.
